docs/README.md

```diff
@@ -0,0 +1,51 @@
+# Streaming guide
+
+Want to stream but don't know where to start? We've got you covered! This guide is designed to help get you started with streaming. The only assumed knowledge is experience with conference calls and sharing your desktop/app on an app like Teams, Skype or Slack. Please note, you will likely need to spend a couple of hours getting a base setup put together.
+
+## Software
+
+The key piece of software you'll need if streaming directly is [OBS Studio](https://obsproject.com/). OBS Studio (or simply OBS) is a powerful OSS streaming suite, and gives you complete control over building the scene for your viewers. Admittedly, OBS Studio can be a little intimidating for anyone who hasn't worked with video editing software in the past.
+
+- [Installing and configuring OBS for streaming](./obs-install.md)
+- [Installing and configuring the Reactor scene collection](./obs-scenes.md) (in progress)
+
+### Software tl;dr
+
+- Actually, on this one you're going to need to read everything :-)
+
+## Hardware guide
+
+While no special hardware is required, a couple of pieces can help elevate your game. We've put together a [hardware guide](./hardware.md) to help you get started.
+
+### Hardware tl;dr
+
+- Have a good mic
+  - The one in your camera/laptop isn't going to be high quality
+  - Chances are the one in your headset isn't either
+- Consider an upgraded camera
+- Don't be back-lit
+
+## Preparing your session
+
+The final steps to preparing to stream is to set the scene. This includes both your physical space and tone and timber. We have a set of [streaming guidelines](digital-stream-guidance.md) to help you hit the right notes.
```

Comment:
Is this supposed to link to the delivering-your-stream.md doc instead? Looks like the .md referenced got deleted.